CMakeLists.txt

```diff
@@ -93,6 +93,10 @@ if(("${CMAKE_CXX_COMPILER_FRONTEND_VARIANT}" STREQUAL "GNU") OR
   target_link_options(OpenSCAD BEFORE PRIVATE "-v")
 endif()
 
+  find_package (Python REQUIRED COMPONENTS Interpreter Development)
+  target_include_directories(OpenSCAD PRIVATE ${Python_INCLUDE_DIRS})
+  target_link_directories(OpenSCAD PRIVATE ${Python_LIBRARY_DIRS})
+  target_link_libraries(OpenSCAD PRIVATE ${Python_LIBRARIES})
```

Comment:
Two suggestions:
1. Wrap find_package() in a cmake check (e.g. -DENABLE_PYTHON) to enable turning on/off Python support on the cmake cmd-line, and make Python REQUIRED in that case.
2. Set a preprocessor macro (using target_compile_definitions()) if ENABLE_PYTHON was set